charger/phoenix-em-eth.go

```diff
@@ -31,49 +31,24 @@ func init() {
 	registry.Add("phoenix-em-eth", NewPhoenixEMEthFromConfig)
 }
 
-//go:generate go run ../cmd/tools/decorate.go -f decoratePhoenixEMEth -b *PhoenixEMEth -r api.Charger -t "api.Meter,CurrentPower,func() (float64, error)" -t "api.MeterEnergy,TotalEnergy,func() (float64, error)" -t "api.PhaseCurrents,Currents,func() (float64, float64, float64, error)"
+//go:generate go run ../cmd/tools/decorate.go -f decoratePhoenixEMEth -b *PhoenixEMEth -r api.Charger -t "api.Meter,CurrentPower,func() (float64, error)" -t "api.MeterEnergy,TotalEnergy,func() (float64, error)" -t "api.PhaseCurrents,Currents,func() (float64, float64, float64, error)" -t "api.PhaseVoltages,Voltages,func() (float64, float64, float64, error)"
 
 // NewPhoenixEMEthFromConfig creates a Phoenix charger from generic config
 func NewPhoenixEMEthFromConfig(other map[string]interface{}) (api.Charger, error) {
-	cc := struct {
-		modbus.TcpSettings `mapstructure:",squash"`
-		Meter              struct {
-			Power, Energy, Currents bool
-		}
-	}{
-		TcpSettings: modbus.TcpSettings{
-			URI: "192.168.0.8:502", // default
```

Comment:
Warum verschwindet der Default? Breaking Change!

Reply:
Wer sollte die Box mit dieser Adresse nutzen?

Reply:
Das ist der alte Wallbe default

Reply:
Was nutzt uns das?
Bleibt in ~99,99% der Netze eine nicht geroutete / nicht erreichbare Adresse.